api/webhook.go

```diff
@@ -90,6 +90,12 @@ func webhookCreate(w http.ResponseWriter, r *http.Request, t auth.Token) error {
 			return err
 		}
 	}
+	for _, kindName := range webhook.EventFilter.KindNames {
+		if kindName == "webhook.run" {
+			return permission.ErrUnauthorized
```

Comment:
I don't think unauthorized is the best error type for this case. The error is caused not because of a lack of permissions, but because this kind of webhook is not allowed. What do you think of a bad request?